source/administration/config-settings.rst

```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
 | This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Global SMTP Server Timeout
```

Comment:
@isacikgoz, I'm not sure whether my explanation is accurate so please correct as needed. :)